### DISCUSSION TITLE
Suppressing <IPython.core.display.HTML object> with console app

### DISCUSSION THREAD

**Discussion by chris-cortner** [guidance-ai/guidance#598] [Q&A] [answered]
I installed jupyter lab in my environment and now when running a console app, the below is displayed.  Any way to disable this?

<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>
<IPython.core.display.HTML object>

**Comment by slundberg** [comment, marked as answer]
you can create the model object with `echo=False`:
```python
gpt4 = models.OpenAI(
    model="gpt-4-1106-preview",
    echo=False
)
```

**Reply by chris-cortner** [reply]
Thanks for the help!
